### ISSUE TITLE
create documentation for building

### ISSUE THREAD

**Issue by langolierz** [langolierz/r_e_c_u_r#18]

**Comment by leobrowning92** [comment]
I spoke to ian about this, and he mentioned that he would be keen to get a pi and set things up from scratch. I think after a first draft of the build documentation is made this would be an excellent opportunity to test naturally, rather than @langolierz  ( who knows the system very well) trying to anticipate what people will need to know.

**Comment by langolierz** [comment]
good idea - i started the build doc [here](https://github.com/langolierz/r_e_c_u_r/blob/master/build%20documentation.md) but theres more info and pictures to add. perhaps we can do this once we have done issue #16 and issue #20 ?
